snapshot-labs/sx-monorepo
# fix(ui): add choices illustrations for dark mode

### Summary

Add voting type illustration for dark mode.

The illustration should now follow the theme

### How to test

1. Go to an offchain proposal creation page
2. It should show a different illustration for the voting type, depending on your current theme

On light mode:
![Screenshot 2024-02-27 at 20 43 27](https://github.com/snapshot-labs/sx-monorepo/assets/495709/946379dc-23c2-4ebb-9c58-012d5f40c903)


On dark mode:
![Screenshot 2024-02-27 at 23 18 29](https://github.com/snapshot-labs/sx-monorepo/assets/495709/31ce024c-d626-4764-a4fc-8f9387efe32b)
